documentation/airport-guides/egll.md

```diff
@@ -0,0 +1,67 @@
+
+Heathrow Airport serves the majority of London air travel, as well as the majority of southern Britain. It is located 14 miles to the West of London's City Center. It is one of the six london airports servicing London (Along with City, Stansted, Luton, Southend and Gatwick). 
+
+Heathrow airport started in 1929 as a small airfield in the hamlet of Heathrow. The development of Heathrow picked up pace in 1944 as a stopover location for long-distance military aircraft, however, by the time it neared completion WW2 had already finished, and the British government continued to develop the airport for civil use. It was opened on 25 March 1946 named 'London Airport', and renamed to 'Heathrow Airport' in 1966. 
+
+### Statistics
+
+78,000,000 passengers in 2017
+474,033 aircraft movement in 2017
+
+[Source](https://www.heathrow.com/company/company-news-and-information/company-information/facts-and-figures)
+
+### Airspace
+
+London has a polygonal airspace serving all six London's airports. More info on the London TMA can be found in [this NATS chart](http://www.ead.eurocontrol.int/eadbasic/pamslight-E89151B3C539313B4E86AD7E95B6927A/7FE5QZZF3FXUS/EN/Charts/ENR/AIRAC/EG_ENR_6_2_1_11_en_2018-03-29.pdf). 
+
+### Standard Operations
+
+###### Runway usage
+
+Heathrow Airport has two runways, 09L/27R and 09R/27L. All four runway approaches are certified for ILS operations, however, Heathrow commonly operates in so called Segregated Mode, meaning one runway is used for departures whilst the other is used for arrivals. Under Westerly operations (landing and departing off 27's), 27L is usually used for departures whilst 27R is used for arrivals, although this configuration is sometimes swapped for noise reduction purposes. In easterly configuration, runway 09L is always the arriving runway, whilst 09R is the departing runway. Westerly operations are generally prefered due to Noise pollution. 
+
+### Instrument Procedures
+
+###### Departures
+
+Departing aircraft usually take off from Heathrow's designated departure runway, and fly their assigned SID at 6000ft before climbing to cruise. 
+
+###### Arrivals
+
+Inbound aircraft are usually vectored to, or follow one of the STARS ending at one of four holding points (Bovingdon: BNN, Biggin Hill: BIG, Lambourne: LAM, Ockham: OCK). They are then brought out of the holds by ATC and merged into a single approach line into Heathrow airport, with separation sometimes as small as 2.5NM. 
+
+### Other airports of interest
+
+##### EGKK
+Gatwick Airport
+
+Gatwick Airport is the second busiest airport in the UK, located near Crawley, West Sussex. It has two asphalt runways, 08L/26R and 08R/26L measuring 8415ft and 10879ft in length respectively.
```

Comment:
Mention that 08L/26R is not used due to the close proximity of both runways, making Gatwick the busiest single-runway airport in the world (I believe, might be worth double-checking that)

Reply:
Gatwick is second, first is Mumbai